pex/resolve/requirement_configuration.py

```diff
@@ -24,8 +24,12 @@ class RequirementConfiguration(object):
     requirement_files = attr.ib(default=None)  # type: Optional[Iterable[str]]
     constraint_files = attr.ib(default=None)  # type: Optional[Iterable[str]]
 
-    def parse_requirements(self, network_configuration=None):
-        # type: (Optional[NetworkConfiguration]) -> Iterable[ParsedRequirement]
+    def parse_requirements(
+        self,
+        network_configuration=None,  # type: Optional[NetworkConfiguration]
+        fallback_requirements=None,  # type: Optional[Iterable[str]]
```

Comment:
`RequirementConfiguration` is used in several places throughout the codebase, so moving `fallback_requirements` into a property on the object seemed incorrect. Lmk if there's a better factoring I should use.